ci/bamboo/setup-dependencies-linux.sh

```diff
@@ -9,37 +9,3 @@ if [ -z "${USER}" ]; then
     USER="docker"
 fi
 export USER
-
-# Setup compiler
```

Comment:
These changes reflect a change in CI from using ubuntu:14.04 base image to numenta-dev-default:0.5.0